frontend/src/components/ConfigForm.vue

```diff
@@ -84,12 +137,36 @@ export default {
     return {
       appConfig: "",
       appConfigStep: 1,
+      step1Valid: false,
+      step2Valid: false,
+      showPw: false,
       commonServices: [
         { text: "Common Messaging Service", value: "cmsg" },
         { text: "Document Management Service", value: "dms" },
         { text: "Document Generation Service", value: "dgen", disabled: true }
       ],
-      userAppCfg: this.$store.state.userAppCfg
+      userAppCfg: this.$store.state.userAppCfg,
+      applicationAcronymRules: [
+        v => !!v || "Acroynm is required",
+        v => v.length <= 30 || "Acroynm must be 10 characters or less",
```

Comment:
Length comparison should be 10 here.
v => v.length <= 10 || "Acroynm must be 10 characters or less",

Reply:
There might be an app that has a acronym longer than 10 characters in existence right now I’m not sure that 10 is our max length but it might be long enough to cover off all existing app acronyms. I’ll go into IRS and do a quick check

Reply:
There might be an app that has a acronym longer than 10 characters in existence right now I’m not sure that 10 is our max length but it might be long enough to cover off all existing app acronyms. I’ll go into IRS and do a quick check

Reply:
@parc-jason right, thanks for catching that discrepancy.
@matthewhall78 yeah I'd actually arbitrarily selected 10, need to find the real max length as you mentioned.

Reply:
currently this is the longest app name:
PRIVATE PROPERTY DB

Reply:
also IRS acronym list has some that contain lowercase, spaces, numbers and these non-alpha numerics:
 ().-_
for example
  ADAM (WILDFIRE) 
  SMARTTOOL 2.0

i think apps that have included there api version in the acronym made a mistake in doing so, since they will need to create a new IRS entry when a new version of the api is released.  so I don’t what to allow service client names or app acronyn names that end in V1 or 2.0.  but I dont see an issue with numbers being used like in app “A4CA”.

so we may need to add a backlog item to look at these edge cases. for now the stricter rule we have is what i want to go with.

Reply:
Yeah restricting newly created ones to a standard is best in my opinion.